src/core_plugins/metrics/server/lib/vis_data/get_annotations.js

```diff
@@ -39,13 +39,13 @@ export default async (req, panel) => {
 
       bodies.push({
         index: indexPattern,
-        ignore: [404],
-        timeout: '90s',
-        requestTimeout: 90000,
+        //ignore: [404],
```

Comment:
@spalger apparently there is some client side `ignore` parameter in some libraries to ignore the response. But I can't find any documentation for that for `elasticsearch.js`. Does this exist at all?